src/NextGen/DonationForm/Blocks/DonationFormBlock/resources/app/utilities/getFormDesign.ts

```diff
@@ -0,0 +1,10 @@
+import {FormDesign} from '@givewp/forms/types';
+
+/**
+ * Get the active form design
+ *
+ * @unreleased
+ */
+export default function getFormDesign(): FormDesign {
```

Comment:
I'm a little nervous about abstracting the long-form path to getting the design. The path itself is critical, and if we abstract it then it won't feel as much to us to be the case. This obviously wouldn't be used outside of core.

Reply:
My point here is to keep the number of ways of accessing node templates reduced to have a consistent (and shared with 3rd parties) DX.